scripts/software/apps.sh

```diff
@@ -10,8 +10,7 @@ function check {
 	key="app_$name"
 
 	if command -v "$name" > /dev/null; then
-
-		echo VALUE BAS "$key" \"$($name $version 2>&1 | head -n1)\" GROUP "apps"
+		echo VALUE BAS "$key" \"$($name $version 2>&1 | sed '/./q; d')\" GROUP "apps"
```

Comment:
@drazioti As I understand it, you changed `head -n1` to `-n2` to support some programs with version outputs that start with an empty line. But in other cases, the second lines contain copyright and other unnecessary stuff.
So I changed back to get only the first line but skip the empty lines.

If this was not the reason, please correct me.

Reply:
Now, after my change (`sed '/./q; d'`),  it works, this is my output:

...
app_perl: This is perl 5, version 22, subversion 1 (v5.22.1) built for x86_64-linux-gnu-thread-multi
app_nmap: Nmap version 7.01 ( https://nmap.org )**
app_nano: GNU nano, version 2.5.3
...